Runner.Tests/ProgramTests.cs

```diff
@@ -11,5 +11,11 @@ public async Task Main_StartupWorks()
             // This is just a quick dumb test for DI
             await Program.Main(new string[0]);
         }
+
+        [Test]
+        public async Task Main_DocumentationGeneration()
+        {
+            await Program.Main(new string[] { "generate-document" });
```

Comment:
What does this test? How does this verify that everything went just fine?

Reply:
Ah, this was a proper töhötys. I improved this by checking that the files are actually generated and have some content.